src/connectors/audius.ts

```diff
@@ -0,0 +1,16 @@
+export {};
+
+Connector.playerSelector = '._playBarContentWrapper_1sp1c_10';
```

Comment:
These look like they will mutate on each rebuild of the site. Anything else we can use to bind too?